plans/chew-large-datasets/test/ipfs_add_defaults.go

```diff
@@ -3,33 +3,54 @@ package test
 import (
 	"context"
 	"fmt"
+	"os"
 
 	files "github.com/ipfs/go-ipfs-files"
 	utils "github.com/ipfs/testground/plans/chew-large-datasets/utils"
+	"github.com/ipfs/testground/sdk/iptb"
 	"github.com/ipfs/testground/sdk/runtime"
 )
 
 // IpfsAddDefaults IPFS Add Defaults Test
-func IpfsAddDefaults(runenv *runtime.RunEnv) {
+type IpfsAddDefaults struct{}
```

Comment:
Not super thrilled by this new approach. If the code that belongs to the test case is the same and the only thing that differs is how you get the IPFS instance (through the library or through go-ipfs-api linked to a daemon), I suggest that you break the logic of getting an IPFS instance into a separate file (the same way we do in js-ipfs when testing js-ipfs Core and js-ipfs through the js-ipfs-api)

Reply:
We checked in person.

@hacdias will improve the current approach to be more symmetric. This will also be great to see how the 2 apis differ so much and how they are hard to use in their own ways. One day, the "Execute" will be the same for both CoreAPI and daemon modes.